database.md

```diff
@@ -0,0 +1,36 @@
+# Database
+
+## Requirements
+Choose a Postgres client, [pgAdmin](https://www.pgadmin.org/) is the most full featured, intelliJ also has a built in plugin.
+
+### Connecting to Local Database
```

Comment:
This and all the following headers should just be `##`, two hashes, instead of `###`, right?  These sections aren't sub-sections bellow requirements, they are siblings at the same level.  Or were you thinking otherwise.